docs/reference/commandline/service_create.md

```diff
@@ -136,6 +136,130 @@ $ docker service create \
+| **source**            |                   | Alias for `src`.
+| **dst**               | yes               | Mount path inside the container, for example `/some/path/in/container/`. If the path does not exist in the container's filesystem, a directory is created at the specified location before the volume or bind-mount is mounted.
+| **destination**       |                   | Alias for `dst`.
+| **target**            |                   | Alias for `dst`.
+| **readonly**          |                   | Mounts the bind or volume read-only in the container. A value is optional, and can be either `true` (or `1`), or `false` (or `0`). If this option is provided without a value, then `true` is used as a default (`readonly=true` is equivalent to `readonly`).
+| **ro**                |                   | Alias for `readonly`.
+
+The following options can only be used for bind-mounts (`type=bind`);
+
+
+| Option                | Description
+|:----------------------|:--------------------------------------------------------------------------------------------------------------------
+| **bind-propagation**  | Bind propagation options to set on the mount at runtime. Valid options are `shared`, `slave`, `private`, `rshared`, `rslave`, and `rprivate`. Defaults to `rprivate` if not specified. For volumes, bind propagation is not configurable, and also defaults to `rprivate`.
+
+
+The following options can only be used for named volumes (`type=volume`);
+
+| Option                | Description
+|:----------------------|:--------------------------------------------------------------------------------------------------------------------
+| **volume-driver**     | Name of the volume-driver plugin to use for the volume. If this option is not provided, the default (`"local"`) volume driver is used to create the volume if it does not exist.
+| **volume-label**      | Custom metadata ("labels") to apply to the volume upon creation. Labels are provided as comma-separated list of key/value pairs, for example, `volume-label=hello=world`. For more information about labels, refer to [apply custom metadata](../../userguide/labels-custom-metadata.md).
+| **volume-nocopy**     | Disables propagating the volume by copying files from the container's filesystem to the volume. By default, if an empty volume is attached to a container, the volume is propagated with the files and directories that are present at the mount-path (`dst`) inside the container. Setting this option disables that behavior, and mounts the volume, even if it's empty. This option takes an optional value, and can be either `true` (or `1`), or `false` (or `0`). If this option is provided without a value, then `true` is used as a default (`volume-nocopy=true` is equivalent to `volume-nocopy`).<br /><br />Note that docker does *not* copy files from the container's filesystem for bind-mounts (`type=bind`). Bind-mounts always use the files/directories from the host.
```

Comment:
This is pretty dense. Maybe we could do something similar to the suggestion for `readonly` above:

By default, if you attach an empty volume is to a container, the Engine propagates the files and directories that are present at the mount-path (`dst`) inside the container into the volume. Set `volume-nocopy` to disables copying files from the container's filesystem to the volume and mount the empty volume.

Value is optional
`true` or `1`: default ...